### ISSUE TITLE
Room settings: color pallette

### ISSUE THREAD

**Issue by chrisrohn** [dubtrack/www-dubtrack-fm#4]
The Pink and cyan is eye catching and all, but it would be great if you provided a dozen varieties of hot/cool color combinations for room owners to select that would better compliment the backgrounds we have in our rooms. Better yet - give us a color wheel to pick the two colors, depending on how much you'd like us to be able to customize.

**Comment by andrewtibbetts** [comment]
-1
2¢: Never let the user control your branding. Let them create browser extensions to change stuff like that. The colors (and possibly the over-flatness of elements) just need to be tightened up, strategized a bit.

**Comment by chrisrohn** [comment]
The color wheel would be letting go of control, and I understand if you wouldn't want to do that.  Having 6-12 pre-determined room color choices would be under full control of dubtrack.  The pink and cyan combination is very intense.

**Comment by andrewtibbetts** [comment]
I agree that the pink+cyan accents are too much. But I still don't think user control over them is the answer—just a redesign.
